cmd/clusterctl/client/cluster/client.go

```diff
@@ -102,6 +102,7 @@ type clusterClient struct {
 	repositoryClientFactory RepositoryClientFactory
 	pollImmediateWaiter     PollImmediateWaiter
 	processor               yaml.Processor
+	isDryrun                bool
```

Comment:
I'm a little debated about having `isDryrun` as a field in the `clusterClient`, because this is an option that applies only to the `Move` command.
Is there a particular reason for this approach? or should we consider to scope down the change to the single command?